examples/kubecondemo/vela.yaml

```diff
@@ -0,0 +1,80 @@
+version: "1.0-alpha.1"
+name: lab3
+
+services:
+  database:
+    type: rds
+    name: alibabaRds
+
+  data-api:
+    type: webservice
```

Comment:
The webservice config isn't aligned with the current CUE template in upstream. Could you upload your webservice template here too?